group-actors/src/main/java/org/sunbird/service/impl/SearchActivityServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package org.sunbird.service.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sunbird.models.ActivitySearchRequestConfig;
+import org.sunbird.service.SearchActivityService;
+import org.sunbird.util.ActivitySearchRequestGenerator;
+import org.sunbird.util.HttpClientUtil;
+import org.sunbird.util.JsonKey;
+
+public class SearchActivityServiceImpl implements SearchActivityService {
+  Logger logger = LoggerFactory.getLogger(SearchActivityServiceImpl.class);
+
+  private static SearchActivityService searchActivityService = null;
+  private static ObjectMapper objectMapper = new ObjectMapper();
+
+  public static SearchActivityService getInstance() {
```

Comment:
@sknirmalkar89 - why do we need a singleton object for SearchActivityService? why cant we have a class with static methods instead?